### DISCUSSION TITLE
Change profile from toolbar

### DISCUSSION THREAD

**Discussion by ghalymt** [SubtitleEdit/subtitleedit#5828] [Ideas]
Can we have "profile" in toolbar, something like frame rate and encoding.
Thank you.

**Comment by LeonCheung** [comment]
Hey, actually, you can click the lower right corner of the  status bar to switch among them (the very bottom of SE window, where displays the current profile name)

**Reply by ghalymt** [reply]
Thank you for that, did not know that it was there.
